src/downloadmanager.cpp

```diff
@@ -164,22 +163,55 @@ bool DownloadManager::moveMyAppPackage(QString path)
     return true;
 }
 
+bool DownloadManager::isMyApp(const QString &targetPath) const
+{
+    return QFileInfo(targetPath).completeSuffix() == QLatin1Literal("myapp");
+}
+
+bool DownloadManager::updatePlatformTransfers(const QString &targetPath) const
+{
+    return !isMyApp(targetPath);
+}
+
+void DownloadManager::finishMyAppDownload(const QString &targetPath) const
+{
+    Q_ASSERT(isMyApp(targetPath));
+
+    QString rootNameSpace("com.aptoide.partners%1");
+    QString aptoideSupport = rootNameSpace.arg(QLatin1Literal(".AptoideJollaSupport"));
+#if CPU_ARM
```

Comment:
Most likely this will bite back. Regrdless of that Store tablet is not actually a store for tablet rather a store that provides x86 apks.